e2e_playwright/st_context_test.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022-2025)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+from playwright.sync_api import Page, expect
+
+
+@pytest.mark.browser_context_args(timezone_id="Europe/Berlin")
+def test_timezone_(app: Page):
+    """Test that the timezone is correctly set."""
+    expect(app.get_by_test_id("stMarkdown").nth(0)).to_have_text("Europe/Berlin")
```

Comment:
We have added the `expect_markdown` utils function recently, and I think its the preferred way of checking for a markdown text on page instead of using index-based access, e.g. `.nth(0)`.

Reply:
We also added `expect_prefixed_markdown` if you want to test it more specifically. E.g. you could add "Timezone name:" and `Timezone offset:` to the `st.write` statements above